companion/discord_bot/bot.py

```
@@ -0,0 +1,435 @@
+            self.logger.warning("Rate limiting will be degraded (fail-open)")
+            # Set to None so checks will gracefully degrade
+            self.redis_client = None
+
+        # Setup commands (slash commands)
+        setup_commands(self, self.gateway_url)
+
+        # Sync slash commands globally (or per guild if preferred)
+        try:
+            await self.tree.sync()
+            self.logger.info("Slash commands synchronized")
+        except Exception as e:
```

Ruff (0.13.2)

Do not catch blind exception: `Exception`

(BLE001)